### DISCUSSION TITLE
How do I search for attached records

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#976] [FAQs]
## Searching for records attached in a field in another module

1. Create a group or search for the records that you want to see the attachments to. Then highlight the records you want to search for.
 
![image](https://github.com/fieldmuseum/EMu-Documentation/assets/702709/d54573d0-afd2-4136-8579-83f44f2a5b6d)

2. Open the module with the attachment field in it.
3. Drag the highlighted records into the field using the "Drag Selected Icon"

![image](https://github.com/fieldmuseum/EMu-Documentation/assets/702709/72f8de74-34cd-47c7-ae2c-cdae819c16ab)

4. Click the search binoculars

**Comment by magpiedin** [comment]
For another example -- searching for Sites attached to a specific set of Narratives:

1) In Sites / search-mode, go to the 'Narratives' tab / search-field.  
..if you don't see the Narratives tab in Sites, [try:  View > Attachments > Options > Show Attachments > Show Search Tabs](https://github.com/fieldmuseum/EMu-Documentation/discussions/403#discussioncomment-2473447)

  - <img width="475" alt="esites_NarrTab" src="https://github.com/user-attachments/assets/0464248a-320d-4ebb-b82e-a1abc5355b1e" />


2) Click that tab's green "+" and a pop-up Narratives window should appear.
...Retrieve and select all Narratives you need, and click that Narrative window's green "+" button (upper right)

  - <img width="475" alt="enarr_attachBack" src="https://github.com/user-attachments/assets/1f794ce7-1207-43b0-a8de-65956f1d522e" />


3) This will attach those Narratives to the query in the Sites module. 
Click the binoculars/run the search in Sites (or add other search criteria as needed)

  - <img width="475" alt="esites_search" src="https://github.com/user-attachments/assets/f257bbf8-fa93-4627-90cf-bc0dc06136a6" />
